hedera-node/docs/design/services/smart-contract-service/evm-transaction-response-codes.md

```diff
@@ -0,0 +1,120 @@
+# EVM Transaction Response Codes in Hedera
+
+When executing a transaction processed by the EVM (i.e., EVM Transaction) using `ContractCreate`, `ContractCall`, `EthereumTransaction`, and `ContractCallLocal` in Hedera, different response codes may be present for the top-level status after the transaction is finished. 
+These responses are found in the transaction receipt. This document's goal is to list the possible EVM transaction type-specific status codes for failure (i.e. different from SUCCESS).
+
+## Propagated EVM Transaction Response Codes
+These response codes can appear for `ContractCall`, `ContractCreate`, and `EthereumTransaction`.
+It is possible for `ContractCreate` and `ContractCall` to create child transactions, and if one of them fails with a special revert or halt reason, the status is propagated to the top-level status of the transaction.
+`ContractCall` creates child transactions by executing logic that results in another call to a function/contract.
+`ContractCreate` creates child transactions by having logic containing calls in the constructor.
```

Comment:
Do you want to make these two lines part of a bulleted list?  And add a line for `EthereumTransaction`.

Reply:
Yep, I think that would make it more clear. Done.